README.md

```diff
@@ -3,36 +3,41 @@
 </p>
 
 # Requirements 🧾
-- Discord Bot (https://discord.com/developers/applications)
-- Python 3.6 or above (https://www.python.org/downloads)
+- Discord developer panel: (https://discord.com/developers/applications)
+- Python 3.6 or above: (https://www.python.org/downloads)
   - Recommended version (3.10.2): https://www.python.org/downloads/release/python-3102/
-- Discord server that you own
-  - Settings -> Enable Community (if you haven't already)
+- Discord server that you own.
+  - `Settings` -> `Enable Community` (if you haven't already)
 
-## Video tutorial 📼
+## Video Tutorial 📼
 If you need visual help on how to do this, then [click here and watch this video](https://i.alexflipnote.dev/7TzozoL.mp4)
 
 ## Usage ✨
-1. Download and install Python if you haven't already
+1. Download and install [Python](https://discord.com/developers/applications) if you haven't already.
```

Comment:
Didn't know Discord developer URL was a Python download link lmao. Should maybe make it use the recommended URL. (3.10.2)

Reply:
How embarrassing lol. I will update this.